engine/security/trust/trust_sandbox.md

```diff
@@ -67,46 +68,74 @@ the `trustsandbox` container, the Notary server, and the Registry server.
         $ mkdir trustsandbox
         $ cd trustsandbox
 
-2. Create a file called `docker-compose.yml` with your favorite editor.  For example, using vim:
+2. Clone the official `notary` repository:
+
+        $ git clone git@github.com:notaryproject/notary.git
+
+3. Create a file called `docker-compose.yml` with your favorite editor.  For example, using vim:
 
         $ touch docker-compose.yml
         $ vim docker-compose.yml
 
-3. Add the following to the new file.
+4. Add the following to the new file.
 
         version: "2"
         services:
-          notaryserver:
-            image: dockersecurity/notary_autobuilds:server-v0.5.1
-            volumes:
-              - notarycerts:/var/lib/notary/fixtures
+          server:
+            image: notary:server-0.6.1-2
+            container_name: notaryserver
```

Comment:
Unfortunately, the file at [fixtures/server-config-local.json](https://github.com/notaryproject/notary/blob/master/fixtures/server-config-local.json#L9) that I am using to configure the image uses the host `notarysigner`. Using the aliases, I ensure we don't have to change anything else about the configuration in a hacky way 😄 

And if I keep the alias in `notarysigner`, then I would keep it for `notaryserver` as well, otherwise, it would look a bit weird.